PoshSSDTBuildDeploy/Functions/InstallMicrosoftDataToolsMsBuild.ps1

```diff
@@ -1,16 +1,15 @@
 
 
 function Install-MicrosoftDataToolsMSBuild {
-
-    param ( [string] $WorkingFolder
-        , [string] $DataToolsMsBuildPackageVersion
+    [cmdletbinding()]
```

Comment:
Added this so you can use the verbose switch at a higher level hence the removal on the verbose switches on 12 & 13.